### ISSUE TITLE
Smoothnet input during training and inference.

### ISSUE THREAD

**Issue by winshot-thu** [cure-lab/SmoothNet#8]
Hi, Ailing and Xuan,
Thanks for your great works! I have some questions:

1. The project Github page says, 'Due to the temporal-only network without spatial modelings, SmoothNet is trained on 3D position representations only, and can be tested on 2D, 3D, and 6D representations, respectively.', which means  3D, 2D, and SMPL models are trained with their corresponding 3D keypoints, right? e.g. PARE SMPL model is trained by PARE 3D pred and g.t. During the inference, we only need input PARE SMPL pred.

2.  The prediction distribution of PARE is inconsistent with SPIN. Is it unreasonable to initialize PARE's Smoothnet by using SPIN's checkpoint？ See github page table "SMPL Results" row 5.
<img width="944" alt="image" src="https://user-images.githubusercontent.com/100033782/182173917-169621b8-3936-41f5-9564-6c97feacb188.png">

3. The code in lib/models/smoothnet looks like the base version of SmoothNet (Only position refinement, no velocity and acceleration). Where is the motion-aware SmoothNet? 

I may have misunderstood your work because of  limited time. I hope your reply.

**Comment by ailingzengzzz** [comment]
Hi @winshot-thu,

Thanks for your kind feedback!
1. This sentence means SmoothNet has a strong generalization ability across modalities. Besides, it can also generalize well across backbones (e.g., train on SPIN backbones and test on PARE backbones). Thus, in several test settings under different datasets, backbones, and modalities, we simply provide three pre-trained models to cover all of these settings. That means you do not need to train on each backbone. For details, please refer to our experiments in both the main paper and Appendix.
2. Although both spatial and temporal distribution (different MPJPEs and Accels) of PARE is inconsistent with SPIN, SmoothNet simply takes an input sequence on each spatial axis that should be smooth as a strong prior knowledge (thanks to the temporal-only MLP networks can learn this priori well), making it can generalize well. You could also refer to how filters work.
3. This figure indicates that you may need to download pre-trained models from the provided Google drive link. Please refer to the data preparation in the ReadMe.
4. Yes, in this clean version, we only release basic SmoothNet. We will add motion-aware SmoothNet later. Motion-aware SmoothNet will make the training process converge faster than the basic one, and their performance is similar.

**Comment by winshot-thu** [comment]
Wow, thanks for your reply! @ailingzengzzz 
I am inspired by the temporal-only design and I will read your papers and work more carefully~
